### PR TITLE
[TT-9703] feat: Add postgres-aggregate backend option to store only aggregate data

### PR DESCRIPTION
**Pull request by jakub-bochenski**
## Description
See #117 

## Related Issue
#117 

https://tyktech.atlassian.net/browse/TT-9703

## Motivation and Context
See #117 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] Documentation updates or improvements.

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [x] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
     - [ ] I have manually updated the README(s)/documentation accordingly.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/tyk-pump/values.yaml [hunk 1]
```diff
@@ -171,7 +171,7 @@ pump:
   containerPort: 9090
 
   # backend defines the pumps to be created by default, as an array of string.
-  # Supported backends are ["mongo", "postgres", "prometheus","hybrid"]
+  # Supported backends are ["mongo", "postgres", "postgres-aggregate", "prometheus","hybrid"]
```
Comment (buraksekili) [inline components/tyk-pump/values.yaml:174]: One minor request is that even though `postgres-aggregate` looks fine to me, we refer to it as `sql-aggregate` in the docs: https://tyk.io/docs/tyk-pump/tyk-pump-configuration/tyk-pump-dashboard-config/#2-sql-aggregate-pump

Can you please update it to `sql-aggregate` to make the naming compatible with docs?

**Review by buraksekili — COMMENTED** [review]
LGTM with a minor suggestion. Thank you @jakub-bochenski for your contribution and effort.

File: components/tyk-pump/values.yaml (hunk 1 above)
Comment (jakub-bochenski) [reply components/tyk-pump/values.yaml:174]: By your logic the already existing option "postgres" should be called "sql".

I can change it if you really want to, but then the naming will be inconsistent.
There should either be "postgress" and "postgress-aggregate" or "sql" and "sql-aggregate".

Comment (jakub-bochenski) [reply components/tyk-pump/values.yaml:174]: Also the implementation is for postgress specifically, not sql, e.g. 
```
"tyk-pump.pg_connection_string_secret_name"
```

Comment (buraksekili) [reply components/tyk-pump/values.yaml:174]: makes sense! then, let's keep as it is.

**Review by buraksekili — APPROVED** [review]

**Pull request merged**